requirements/edx/github.txt

```diff
@@ -77,7 +77,7 @@ git+https://github.com/edx/lettuce.git@0.2.20.002#egg=lettuce==0.2.20.002
 git+https://github.com/edx/edx-ora2.git@1.4.3#egg=ora2==1.4.3
 -e git+https://github.com/edx/edx-submissions.git@2.0.0#egg=edx-submissions==2.0.0
 git+https://github.com/edx/ease.git@release-2015-07-14#egg=ease==0.1.3
-git+https://github.com/edx/edx-val.git@0.0.13#egg=edxval==0.0.13
+git+https://github.com/edx/edx-val.git@ammar/auto-generated-images-fixes
```

Comment:
@mushtaqak I will merge this edx-val PR and then create a tag and update the course-video-image branch today.